src/plugins/map/MapObject.h

```diff
@@ -206,28 +223,30 @@ class MapObjectMapLoc : public MapObject
 
 	virtual bool CanDisplayObject() const override { return true; }
 
+	bool				  m_isSelected; // exposed directly for imgui reference
+
 private:
 	void UpdateMapLoc();
 	void RemoveMapLoc();
-
 	void UpdateText();
 
 	bool                  m_isCreatedFromDefaultLoc;
 	bool                  m_initialized = false;
 	int                   m_index = 1;
 	std::string           m_tag;
 	std::string           m_labelText;
-	float                 m_lineSize;
-	float                 m_width;
-	MQColor               m_color;
 	std::vector<MapViewLine*> m_lines;
-	float                 m_circleRadius;
-	MQColor               m_circleColor;
 	MapCircle             m_circle;
+	MapLocParams		  m_mapLocParams;
 };
 
+extern std::map<std::string, MapObjectMapLoc*> sLocationsMap;
+extern std::vector<MapObjectMapLoc*> sMapLocs;
```

Comment:
s prefix means static a.k.a. file local. making these extern means they can't possibly be static anymore.

Reply:
good call, was named before moved here